<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LifDisplayWidget</class>
 <widget class="QWidget" name="LifDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="LifTracePlot" name="lifTracePlot" native="true">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="LifSlicePlot" name="timeSlicePlot" native="true">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="LifSlicePlot" name="freqSlicePlot" native="true">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>8</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="lifSpectrogram" native="true">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>235</y>
     <width>840</width>
     <height>335</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LifTracePlot</class>
   <extends>QWidget</extends>
   <header>liftraceplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LifSlicePlot</class>
   <extends>QWidget</extends>
   <header>lifsliceplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
